src/main/java/book/shop/spring/boot/intro/model/Book.java

```diff
@@ -63,6 +63,6 @@ public class Book {
     @EqualsAndHashCode.Exclude
     private Set<Category> categories = new HashSet<>();
 
-    @Column(nullable = false)
-    private Boolean deleted;
+    @Column(columnDefinition = "TINYINT(1)", nullable = false)
+    private boolean deleted = false;
```

Comment:
when creating boolean type usually add "is.." for such type of fields
```suggestion
    private boolean isDeleted = false;
```